### PR TITLE
[10.0][MIG][partner_identification_gln] mig partner identification gln mgs

### PR DESCRIPTION
**Pull request by rvalyi**
attempt to supersede https://github.com/OCA/partner-contact/pull/446 with rebasing on 10.0 to see if it fixes Travis Lint tests...

### CODE REVIEW

**Comment by rvalyi** [comment]
cc @rousseldenis @Cedric-Pigeon @GillesTephaneMeyomesse 
@rousseldenis you approved the old PR, can we merge this one that is rebased on 10.0 and pass the tests? It is OK for me.

**Review by Cedric-Pigeon — APPROVED** [review]

**Comment by rousseldenis** [comment]
Ok if passing travis

**Comment by rvalyi** [comment]
This PR was long approved in its previous version  #446. I tested,m,m approved and just did a rebased here to get the tests passing again, so merging...

File: partner_identification_gln/__manifest__.py
Comment (nikul-serpentcs) [inline partner_identification_gln/__manifest__.py:13]: @rvalyi URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: partner_identification_gln/data/partner_identification_gln.xml
Comment (nikul-serpentcs) [inline partner_identification_gln/data/partner_identification_gln.xml:15]: add new blank line

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

**Comment by BT-mgomez** [comment]
Hello @nikul-serpentcs @rvalyi @rousseldenis

What is the pending status of this PR? will be merged to 10.0 ?
In another, I can not find a migration to version 12. Do you know if someone is working on it, or can I do it? or has this functionality be imported as core functionality in Odoo v11 or v12 ?
Neither I can find another module in OCA for this functionality, and as far as I understood this is a key functionality for Germany for some market operations in some companies.

Thanks.

File: partner_identification_gln/__init__.py
Comment (NL66278) [inline partner_identification_gln/__init__.py:2]: Remove copyright line from __init__.py files.

File: partner_identification_gln/__manifest__.py
Comment (NL66278) [reply partner_identification_gln/__manifest__.py:13]: I agree with @nikul-serpentcs here.

**Review by NL66278 — CHANGES_REQUESTED** [review]
Only few changes needed before this can be merged. @rvalyi can you take care of the comments by @nikul-serpentcs and me, so we can move forward with this very usefull module?

**Review by rousseldenis — APPROVED** [review]
Code review apart existing comments.

**Comment by rousseldenis** [comment]
@rvalyi Could you edit PR title to make it more readable ?

**Comment by NL66278** [comment]
@rvalyi Could you please take up the few changes requested, so we can merge this? Thanx in advance!

**Pull request merged**
